scripts/gpp_stub.js

```diff
@@ -0,0 +1,150 @@
+(window.__gpp_addFrame = function (e) {
+  if (!window.frames[e])
+    if (document.body) {
+      var t = document.createElement("iframe");
+      (t.style.cssText = "display:none"),
+        (t.name = e),
+        document.body.appendChild(t);
+    } else window.setTimeout(window.__gpp_addFrame, 10, e);
+}),
+  (window.__gpp_stub = function () {
+    var e = arguments;
+    if (
+      ((__gpp.queue = __gpp.queue || []),
+      (__gpp.events = __gpp.events || []),
+      !e.length || (1 == e.length && "queue" == e[0]))
+    )
+      return __gpp.queue;
+    if (1 == e.length && "events" == e[0]) return __gpp.events;
+    var t = e[0],
+      p = e.length > 1 ? e[1] : null,
+      s = e.length > 2 ? e[2] : null;
+    if ("ping" === t)
+      p(
+        {
+          gppVersion: "1.1",
+          cmpStatus: "stub",
+          cmpDisplayStatus: "hidden",
+          signalStatus: "not ready",
+          supportedAPIs: [
+            "2:tcfeuv2",
+            "5:tcfcav1",
+            "6:uspv1",
+            "7:usnatv1",
+            "8:uscav1",
+            "9:usvav1",
+            "10:uscov1",
+            "11:usutv1",
+            "12:usctv1",
+          ],
+          cmpId: 0,
+          sectionList: [],
+          applicableSections: [],
+          gppString: "",
+          parsedSections: {},
+        },
+        !0,
+      );
+    else if ("addEventListener" === t) {
+      "lastId" in __gpp || (__gpp.lastId = 0), __gpp.lastId++;
+      var n = __gpp.lastId;
+      __gpp.events.push({ id: n, callback: p, parameter: s }),
+        p(
+          {
+            eventName: "listenerRegistered",
+            listenerId: n,
+            data: !0,
+            pingData: {
+              gppVersion: "1.1",
+              cmpStatus: "stub",
+              cmpDisplayStatus: "hidden",
+              signalStatus: "not ready",
+              supportedAPIs: [
+                "2:tcfeuv2",
+                "5:tcfcav1",
+                "6:uspv1",
+                "7:usnatv1",
+                "8:uscav1",
+                "9:usvav1",
+                "10:uscov1",
+                "11:usutv1",
+                "12:usctv1",
```

Comment:
I think we're going to want to match this list: https://github.com/IABTechLab/iabgpp-es/blob/3.1.4/modules/cmpapi/src/encoder/section/Sections.ts#L15-L30